src/lib/components/PublishButton.js

```diff
@@ -15,6 +15,13 @@ import { submitAction } from '../state/actions';
 import { FORM_PUBLISHING } from '../state/types';
 import { toCapitalCase } from '../utils';
 
+const requiredFieldsForValidDraft = [
+  'title',
+  'resource_type',
+  'publication_date',
+  'creators',
+];
+
```

Comment:
I don't like to have the required fields hardcoded here ... but could't find a better solution.
Is there another way of knowing if the draft is valid?